### DISCUSSION TITLE
Has anyone run performance test or stress test against the Dify library? Our result is pretty bad and no sure if we made something wrong

### DISCUSSION THREAD

**Discussion by jimmyxu1985** [langgenius/dify#3677] [Help]
Hi all, just want to know if anyone run a performance test or stress test aganst Dify library. We have a result around 10 QPS for one pod (1 CPU + 2G memory) which doesn't seem correct.

**Comment by crazywoola** [comment]
Well according to our readme

> Before installing Dify, make sure your machine meets the following minimum system requirements:

> CPU >= 2 Core
> RAM >= 4GB
